frontend/src/components/Retweet.jsx

```diff
@@ -82,6 +83,29 @@ const Retweet = () => {
         setPreviewLoading(false);
       });
   };
+
+  const unlikeTweet = async () => {
+    if (tweetIdRef.current.value.length < 5) {
+      toast.warn("Please enter valid Tweet ID!");
+      return;
+    }
+    setLikeLoading(true);
+    fetch(backendUrl + "/unlike?id=" + tweetId).then((res)=>res.json())
+      .then((res) => {
+        if (res.data) {
+          toast.success("Tweet Unliked!");
+        } else if (res.errors) {
```

Comment:
can you please change the `res.errors` to `res.error` in line 97 and 26.

Reply:
Sure